docker_templates/templates/docker_images/ros1_bridge_entrypoint.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 set -e
 
+unset ROS_DISTRO
 # setup ros1 environment
-source "/opt/ros/$ROS_DISTRO/setup.bash"
+source "/opt/ros/$ROS1_DISTRO/setup.bash"
 
```

Comment:
Not sure if it would affect how the ros bridge is loaded, but addind a `unset ROS_DISTRO` here could help silence the warning in the docker logs when using this entrypoint as is. E.g.
"ROS_DISTRO was set to 'melodic' before. Please make sure that the environment does not mix paths from different distributions."

Reply:
Yeah I considered it and couldn't make up my mind :)
I think silencing it would be good. The reason I kept it is that people running the same set of commands on a machine having both ROS1 and ROS2 installed would have the same warning:
```
source /opt/ros/melodic/setup.bash
source /opt/ros/bouncy/setup.bash
```

One could argue that in the case of the entry point we have control over the environment so it doesn't add value to keep the warning as it's not due to "user error"

Reply:
In conclusion I think you're right and there is no real drawback in silencing the warning here: https://github.com/osrf/docker_templates/pull/43/commits/2ba520981e85c9d25a0c4b7683c0cfc98c78b333

Reply:
:+1: